### ISSUE TITLE
Bullet Rails6 compatibility

### ISSUE THREAD

**Issue by callaedwards** [flyerhzm/bullet#451]
It appears bullet isn't updated to work with rails6 beta versions yet:

```
There was an error while trying to load the gem 'bullet'. (Bundler::GemRequireError)
Gem Load Error is: Bullet does not support active_record 6.0.0.beta3 yet
```

will this be available soon with the release of rails6 later this month?

**Comment by jhixson** [comment]
Looks like a 6.0 update was just pushed today. I updated my Gemfile to reference the repo and it worked. I'm sure rubygems will be updated soon too.

**Comment by connorshea** [comment]
@flyerhzm can you push the 6.0.0 version to RubyGems? Thank you for your work on this gem ❤️

**Comment by flyerhzm** [comment]
I just released bullet 6.0.0 which supports rails 6.0.0 although there's a test failure, I'll try to fix it later.

**Comment by connorshea** [comment]
Thanks!
